tests/UnitTests/GT.Core.Tests/GTExperienceLevelServiceTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using FluentAssertions;
+using GT.Core.Services.Impl;
+using GT.Data.Data.GTAppDb.Entities;
+using GT.Data.Repositories.Interfaces;
+using Microsoft.Extensions.Logging;
+using MockQueryable.Moq;
+using Moq;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace GT.Core.Tests
+{
+	public class GTExperienceLevelServiceTests
+	{
+
+		[Theory]
+		[InlineData("NameInDb", true)]
+		[InlineData("NameNotInDb1", false)]
+		[InlineData(null, false)]
+		public async Task ExistsByNameAsync_ExpectedBehaviour_Succeeds(string inputName, bool expected)
```

Comment:
Consider rewriting method name to something clearer.

Suggestion: "ExistsByNameAsync_ReturnsExpectedValue_Succeeds"

Reply:
Good idea. Fix in commit.